packages/stylefire/src/css/build-styles.ts

```diff
@@ -5,7 +5,6 @@ import {
   isTransformProp,
   isTransformOriginProp
 } from './transform-props';
-import prefixer from './prefixer';
```

Comment:
is prefixing happening at some other place now?

Reply:
Good catch, yeah I did add it elsewhere in `build-styles` but must have lost the change at some point